packages/esm-patient-forms-app/src/htmlformentry/html-form-entry-wrapper.component.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import styles from './html-form-entry-wrapper.scss';
+
+interface HtmlFormEntryWrapperProps {
+  closeWorkspace: () => void;
+  src: string;
+}
+
+const HtmlFormEntryWrapper: React.FC<HtmlFormEntryWrapperProps> = ({ closeWorkspace, src }) => {
```

Comment:
So basically all this does is load the O2 app into an iframe and hides the headers and the breadcrumbs